src/contest_tier.py

```
@@ -0,0 +1,70 @@
+    if not value.startswith(prefix):
+        return None
+    return _normalize_tier_key(value.removeprefix(prefix).lstrip("-:"))
+
+
+def _normalize_tier_key(value: str) -> Optional[str]:
+    if value in TIER_DISPLAY:
+        return value
+    return None
+
+
+def normalize_contest_tier(raw_tier: Any) -> Optional[str]:
```

Ruff (0.15.5)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `raw_tier`

(ANN401)